src/test/scala/org/broadinstitute/dsde/firecloud/service/LibraryServiceSpec.scala

```diff
@@ -340,7 +344,24 @@ class LibraryServiceSpec extends FreeSpec with LibraryServiceSupport with Attrib
           AttributeName.withDefaultNS("workspaceId") -> AttributeString(testWorkspace.workspaceId)
         ))
         assertResult(expected) {
-          indexableDocument(w)
+          Await.result(indexableDocument(w, ontologyDao), dur)
+        }
+      }
+    }
+    "with diseaseOntologyID attribute with parents" - {
+      "should generate indexable document with parent info" in {
+        val w = testWorkspace.copy(attributes = Map(
+          AttributeName.withLibraryNS("diseaseOntologyID") -> AttributeString("DOID_9220")
+        ))
+        val expected = Document(testUUID.toString, Map(
+          AttributeName.withLibraryNS("diseaseOntologyID") -> AttributeString("DOID_9220"),
+          AttributeName.withDefaultNS("name") -> AttributeString(testWorkspace.name),
+          AttributeName.withDefaultNS("namespace") -> AttributeString(testWorkspace.namespace),
+          AttributeName.withDefaultNS("workspaceId") -> AttributeString(testWorkspace.workspaceId),
+          AttributeName.withDefaultNS("parents") -> AttributeValueRawJson(ontologyDao.resources.get(0).parents.toJson.compactPrint)
+        ))
+        assertResult(expected) {
+          Await.result(indexableDocument(w, ontologyDao), dur)
```

Comment:
Since we're holding each other to good testing practices, other unit tests I'd like to see:
* case where the diseaseOntologyID returns a node without any parent information
* case where the diseaseOntologyID is not found